src/components/ProfileImg/ProfileImg.jsx

```diff
@@ -0,0 +1,42 @@
+import Avatar from '@mui/material/Avatar';
+import React from 'react';
+import './style.css';
+import { useState, useEffect } from 'react';
+import client from '../../utils/client';
+
+function ProfileImg({ avatar }) {
+  const userId = localStorage.getItem('userId');
```

Comment:
I think it might be a good idea to use @marianphillips's new localStorage module to get the `userId` rather than calling localStorage yourself.

Reply:
great suggestion, sorted now